### PR TITLE
prevent "Job for docker.service canceled" error on startup - v4.3

### PR DESCRIPTION
**Pull request by jim-minter**
```release-note
NONE
```

### CODE REVIEW

**Comment by jim-minter** [comment]
added "work around broken v3.2 build" commit, readding lgtm label

File: pkg/arm/v4/data/master-startup.sh [hunk 1]
```diff
@@ -1,6 +1,10 @@
 #!/bin/bash -ex
 
 if ! grep /var/lib/docker /etc/fstab; then
+{{- if ne .Config.PluginVersion "v4.2" }}
+  systemctl stop docker-cleanup.timer
```
Comment (thekad) [inline pkg/arm/v4/data/master-startup.sh]: this wouldn't interrupt the docker-cleanup process if it's currently running, would it?

**Comment by jim-minter** [comment]
CI broken, merging

**Pull request merged**